.mvn/extensions.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extensions>
```

Comment:
this is a maven extension that allows to override maven settings.xml at project level